### DISCUSSION TITLE
[RFC Call] Agenda - February 1, 2021

### DISCUSSION THREAD

**Discussion by jonathantneal** [withastro/roadmap#93] [Proposal]
# Agenda

There are no items to reach consensus on today. Instead, we will use the time to discuss new features in Astro and open RFC discussions.

- Features recently added to Astro
- RFC discussions in flight.

Then, let’s focus on a few discussions that have interested the community and the core team:

- import assertions (followup of https://github.com/withastro/rfcs/discussions/69)
- improving loops in markup (https://github.com/withastro/rfcs/discussions/94)
- changing link formatting (followup of https://github.com/withastro/rfcs/discussions/79)
- embedding syntaxes; e.g. `raw:html`, like `set:html` but for content (followup of https://github.com/withastro/rfcs/discussions/84)
- knowing static build restrictions (followup of https://github.com/withastro/rfcs/discussions/90)

--- 

:astronaut: Our **Community RFC Call** starts <a href="https://hammertime.djdavid98.art/?t=1643742000"><time datetime="2022-02-01T19:00:00.000Z">Tue, 01 Feb 2022 19:00:00 GMT<time></a>  :astronaut:

This event is open to everyone in the community. Join us to discuss proposals that will improve and enhance Astro.

If there is something that you would like to see discussed, please comment below. We will prioritize pending RFC, but if we have time to spare we will open the call up to get attention on and discuss [other threads and feature requests](https://github.com/withastro/rfcs/discussions).

**Comment by lucemans** [comment]
If any time left, maybe look over [#2519](https://github.com/withastro/astro/pull/2519) ?

Im in call just listenin' (🤷🏻 )

**Reply by FredKSchott** [reply]
Looks good! If we don't get to it, this kind of PR doesn't need agreement in RFC, should be fine to get reviewed directly in that PR!
